server/repository/src/migrations/v2_06_00/add_cancelled_status_to_invoice.rs

```diff
@@ -12,9 +12,12 @@ impl MigrationFragment for Migrate {
                 connection,
                 r#"
                 ALTER TYPE invoice_status ADD VALUE IF NOT EXISTS 'CANCELLED';
+                ALTER TYPE activity_log_type
+                ADD VALUE IF NOT EXISTS
+                    'PRESCRIPTION_STATUS_CANCELLED' AFTER 'PRESCRIPTION_STATUS_VERIFIED';
```

Comment:
Added to existing migration, so might need to manually reset the migration fragment.